packages/progress-bar/src/ProgressBar/ProgressBar.styles.ts

```diff
@@ -7,10 +7,23 @@ import {
   spacing as spacingToken,
 } from '@leafygreen-ui/tokens';
 
-import { Color, Size } from './ProgressBar.types';
+import { AnimationMode, Color, Size } from './ProgressBar.types';
 import { getPercentage } from './ProgressBar.utils';
 
-const progressBarSizeStyles = {
+export const FADEOUT_DURATION = 500;
+
+const opacityAlphaChannels = {
```

Comment:
I'm not that familiar with alpha channels; how did you determine these values?

Reply:
https://github.com/mongodb/leafygreen-ui/blob/2622c191795c40b587c9f61142eb8e9f4c0d2d01/packages/select/src/styleSets.ts#L43